### ISSUE TITLE
Allow users to sign in to manually track content which isn't available through The Lodestone

### ISSUE THREAD

**Issue by JamesDonnelly** [ApkalluFalls/apkallufalls.com#10]
One of the most requested features on Apkallu Falls is the ability to check off content which isn't automatically trackable by The Lodestone. This currently includes Emotes and Chocobo Barding (#8), but in future may be extended to include Orchestrion Rolls, Hairstyles (#9) and whatever else.

There are several roadblocks before this can become a reality:

- [ ] Investigation
  - [ ] What services should be used to sign in?
  - [ ] How can these services be integrated into Apkallu Falls?
- [ ] Database
  - [ ] Create users table
  - [ ] Create emotes table
  - [ ] Create chocobo barding table
- [ ] Website
  - [x] Allow users to sign in
  - [x] Allow users to sign out
  - [x] Create sign in box, visible on all pages
  - [x] Integrate with Twitter
  - [ ] Integrate with ... (services TBD as per above investigation)
  - [ ] Modify list pages to include checkboxes
  - [ ] Make it clear that all user data is private (privacy policy?)

**Comment by JamesDonnelly** [comment]
With regard to "What services should be used to sign in?", I've created a Twitter poll asking users which services they'd like to use: https://twitter.com/ApkalluFalls/status/1009157074759102466.
